build.sbt

```diff
@@ -100,3 +105,18 @@ lazy val exercise_050_cluster_cluster_singleton_akka_bootstrap_discovery_via_con
 lazy val exercise_051_cluster_singleton_akka_bootstrap_discovery_via_akka_dns = project
   .configure(CommonSettings.configure)
   .dependsOn(common % "test->test;compile->compile")
+
+lazy val common_display = project.settings(CommonSettingsDisplay.commonSettings: _*)
```

Comment:
I see that you put a lot of `.jar` files under the `common_display` project. Why not use regular dependencies (declared under `project/Dependencies.scala`)?

They seems to exist on Maven:
```
"com.pi4j" % "pi4j-core" % "1.2"
"com.pi4j" % "pi4j-device" % "1.2"
"com.pi4j" % "pi4j-gpio-extension" % "1.2"
"com.pi4j" % "pi4j-example" % "1.2"
```

Reply:
Yeah, when I added them initially several months ago there was pom files but not jars. Now - I checked that jars are available on maven central - will remove jar from repo